python/cugraph/link_analysis/pagerank.py

```diff
@@ -13,10 +13,12 @@
 
 from cugraph.link_analysis import pagerank_wrapper
 from cugraph.structure.graph import null_check
+import cugraph
 
 
 def pagerank(
-    G, alpha=0.85, personalization=None, max_iter=100, tol=1.0e-5, nstart=None
+    G, alpha=0.85, personalization=None, max_iter=100, tol=1.0e-5, nstart=None,
```

Comment:
Nx default tolerance is 1e-8, does that matter?

Reply:
I need to run a set of benchmarks to determine if there is an benefit to the the lower value